test_util/helpers.py

```diff
@@ -170,6 +170,27 @@ def options(self, *args, **kwargs):
         return self.api_request('OPTIONS', *args, **kwargs)
 
 
+def is_retryable_exception(exception):
+    for ex in [requests.exceptions.ConnectionError, requests.exceptions.Timeout]:
+        if isinstance(exception, ex):
+            return True
+    return False
+
+
+class RetryCommonHttpErrorsMixin:
+    """ Mixin for ApiClientSession so that random disconnects from
+    network instability do not derail entire scripts
+    """
+    def api_request(self, *args, retry_timeout=60, **kwargs):
```

Comment:
I am not sure the unit for `retry_timeout`, could this be added to the method doc or rename to `retry_timeout_UNIT`?